Extension/src/Debugger/configurationProvider.ts

```diff
@@ -188,85 +188,94 @@ class CppConfigurationProvider implements vscode.DebugConfigurationProvider {
 	 * Try to add all missing attributes to the debug configuration being launched.
 	 */
     resolveDebugConfiguration(folder: vscode.WorkspaceFolder | undefined, config: vscode.DebugConfiguration, token?: vscode.CancellationToken): vscode.ProviderResult<vscode.DebugConfiguration> {
-        if (config) {
-            if (config.type === 'cppvsdbg') {
-                // Fail if cppvsdbg type is running on non-Windows
-                if (os.platform() !== 'win32') {
-                    logger.getOutputChannelLogger().showWarningMessage(localize("debugger.not.available", "Debugger of type: '{0}' is only available on Windows. Use type: '{1}' on the current OS platform.", "cppvsdbg", "cppdbg"));
-                    return undefined;
-                }
+        // if config or type is not specified, return null to trigger VS Code to open a configuration file https://github.com/Microsoft/vscode/issues/54213
```

Comment:
use github bug notation and put bug at the front in brackets might make it easierl
Microsoft/vscode#54213